src/main/java/entities/Profile.java

```diff
@@ -0,0 +1,101 @@
+package entities;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class Profile{
+    private String profileName;
+    private LocalDate dob;
+    private String description;
+    private ArrayList<String> socialLinks;
+    private ArrayList<String> sensitiveWords;
+    private ArrayList<String> interests;
+    private ArrayList<String> groups;
+    private ArrayList<User> friends;
+    private ArrayList<User> blockedUsers;
+
```

Comment:
Should we be adding an instance variable that ties the User object to the Profile object?